cpt/runner.py

```diff
@@ -148,7 +149,7 @@ def __init__(self, profile_text, base_profile_text, base_profile_name, reference
                  pip_install=None,
                  config_url=None,
                  printer=None,
-                 upload_dependencies=None,
+                 upload_dependencies=[],
```

Comment:
It is not a good idea to default a list in an argument in Python. The default will be cached between calls. 
Init none or [] in the body. read here: https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html

Reply:
done!